src/components/Accordian/Accordian.tsx

```diff
@@ -0,0 +1,87 @@
+
+import { FC, MouseEvent, SyntheticEvent, useState } from 'react';
+import { classNames } from '../../utils/classNames';
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+//props type
+export interface AccordionProps {
```

Comment:
Backlog: The developer can provide the summary/detail/expandIcon of the component from props.
This component should provide a expand Icon prop to user, so they can customize the icon